docs/rst/use_cases/wan_tcp.rst

```diff
@@ -0,0 +1,256 @@
+.. _use_cases_wan_tcp:
+
+WAN communication over TCP
+==========================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+This tutorial covers the first steps in setting up a WAN DDS communication over TCP.
+More specifically, we will run a DDS Subscriber on a machine deployed on a local ``network A`` and a DDS Publisher on a machine deployed on a local ``network B``.
+Both DDS entities will communicate over the Internet by means of the DDS Router.
+
+.. _warning_lan:
+
+.. warning::
+
+    This tutorial is intended for WAN communication.
+    However, if there is only access to a LAN communication, it is possible to follow the tutorial by changing the DDS Domain Id so that the Subscriber uses the default Domain (``0``) and the Publisher uses DDS Domain ``1``.
+    This way the DDS entities are logically isolated and will not discovery other entities out of their DDS Domain.
+
+The image below describes the scenario presented in this tutorial.
+
+.. figure:: /rst/figures/use_cases/wan_tcp.png
+   :align: center
+
+Several key elements can be observed in it:
+
+1.  **Fast DDS Subscriber** and **Fast DDS Publisher**.
+    The application used for this tutorial is the *BasicHelloWorldExample*.
+    The *BasicHelloWorldExample* is a Fast DDS application that implements a subscriber and a publisher, to which a configuration of basic DDS and QoS parameters such as DDS domain, transport, communication reliability, among others, can be applied.
+
+1.  **DDS Router**.
+    *eProsima DDS Router* is an end-user software application that enables the connection of distributed DDS networks.
+    That is, DDS entities such as publishers and subscriber deployed in one geographic location and using a dedicated local network will be able to communicate with other DDS entities deployed in different geographic areas on their own dedicated local networks as if they were all on the same network through the use of *DDS Router*.
+
+    This example presents two routers that enable Internet communication:
+
+    * *DDS Router Net A*. This is the DDS Router that is deployed on *LAN A*. This way it is possible for the robot to communicate out-of-the-box with an external DDS Router configured as server.
+    * *DDS Router Net B*. It plays the server role in the communication. It will expose a public network address to which client DDS Routers connect to establish communication.
+
```

Comment:
Specify that there is no relation between Publisher/Subscriber and client/server. This has been a common mistake of users using TCP example.
You can also comment that communication goes in both directions.